nomad/csi_endpoint_test.go

```diff
@@ -1981,18 +1980,24 @@ func TestCSI_SerializedControllerRPC(t *testing.T) {
 	var wg sync.WaitGroup
 	wg.Add(3)
 
-	timeCh := make(chan lang.Pair[string, time.Duration])
+	timeCh := make(chan struct {
+		pluginID string
+		dur      time.Duration
+	})
 
 	testFn := func(pluginID string, dur time.Duration) {
 		defer wg.Done()
-		c := NewCSIVolumeEndpoint(srv, nil)
+		c := &CSIVolume{srv: srv, logger: nil}
```

Comment:
This constructor doesn't exist in 1.4 (it was introduced as part of the RPC rate metrics work), so I'm just instantiating this directly.